CONTRIBUTE.md

```diff
@@ -5,7 +5,7 @@ There are many ways to contribute to IPFS development. This document will flesh
 
 ### Help with the design
 
-If you find an error or have a suggestion for the overall design of IPFS, please submit an [Issue](https://github.com/ipfs/ipfs/issues/new).
+If you find an error or have a suggestion for the overall design of IPFS, please submit an [Issue](https://github.com/ipfs/ipfs/issues/new). Note: If you an error with an [implementation](README.md#implementations), file an issue in the respective repository. This repository's focus is the abstracted design.
```

Comment:
almost, the "abstracted design" is over at https://github.com/ipfs/specs -- this is more of a "landing repo" and "big update place"